projects/log4j2/build.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -eu
-# Copyright 2021 Google Inc.
+# Copyright 2024 Google LLC
```

Comment:
No need to update copyright year (can keep the LLC change)

Reply:
Fixed in 11d428e5054012dd9944908fe69dfd99ab6d4166.